config_app/conf/supervisord.conf

```diff
@@ -13,14 +11,6 @@ serverurl=unix:///%(ENV_QUAYDIR)s/config_app/conf/supervisord.sock
 [rpcinterface:supervisor]
 supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
 
-[eventlistener:stdout]
-environment=
-  PYTHONPATH=%(ENV_QUAYDIR)s
-command = supervisor_stdout
-buffer_size = 1024
-events = PROCESS_LOG
-result_handler = supervisor_stdout:event_handler
-
```

Comment:
The config app should have all process go to stdout as well.

Reply:
They should already be:
```
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
stderr_logfile=/dev/stdout
stderr_logfile_maxbytes=0
```